gtda/utils/validation.py

```diff
@@ -174,3 +176,117 @@ def validate_metric_params(metric, metric_params):
                              " parameter. Available metric_params"
                              " are {}".format(param,
                                               available_metric_params[metric]))
+
+
+def check_list_of_images(X, **kwargs):
+    """"Check a list of arrays representing images, by integrating
+    through the input one by one. To pass a test with
+    :param:`kwargs`=``None``, all images x in X must satisfy:
+        - x.ndim >= 2,
+        - all(np.isfinite(x))
+
+    Parameters
+    ----------
+    X : list of ndarray,
+        Each entry of X corresponds to an image.
+
+    kwargs: dict or None, optional, default: ``None``
+        Parameters accepted by
+        :func:`~gtda.utils.validation.check_list_of_arrays`.
+
+    Returns
+    -------
+    X : list of ndarray, as modified by check_array
+
+    """
+    kwargs_default = {'force_same_n_axis':  False,
+                      'force_same_dim': False, 'force_all_finite': True,
+                      'ensure_2d': False, 'allow_nd': True}
+    kwargs_default.update(kwargs)
+    return check_list_of_arrays(X, **kwargs_default)
+
+
+def check_list_of_point_clouds(X, **kwargs):
+    """"Check a list of arrays representing point clouds, by integrating
```

Comment:
Implement same (where applicable) dosctring fixes as for `check_list_of_images`.